src/Snorter_Ubuntu-14.04.sh

```diff
@@ -82,7 +102,24 @@ function snort_install() {
 	echo -ne "\n\t${CYAN}[i] INFO:${NOCOLOR} /var/log/snort and /etc/snort created and configurated.\n\n"
 	sudo /usr/local/bin/snort -V
 	echo -ne "\n\t${GREEN}[+] INFO:${NOCOLOR} ${BOLD}SNORT${NOCOLOR} is successfully installed and configurated!"
-
+	
+	#Installing OpenAppID
+	echo -ne "\n\t${CYAN}[i] INFO:${NOCOLOR} Installing ${BOLD}$OpenAppID${NOCOLOR}.\n\n"
+	cd $HOME/snort_src/
+	wget --no-check-certificate -P $HOME/snort_src/ https://snort.org/downloads/openappid/5411 -O snort-openappid.tar.gz
```

Comment:
+ [x] Done

Also created a new function `install_openappid`, which will be only executed if the `-a` flag is set.